src/unix/mediactrl.cpp

```diff
@@ -35,7 +35,13 @@
 
 #ifdef __WXGTK__
     #include "wx/gtk/private/wrapgtk.h"
-    #include <gdk/gdkx.h>
+    #include "wx/gtk/private/mediactrl.h"
+#endif
+
+#if GST_CHECK_VERSION(1,0,0)
+    typedef guintptr window_id_type;
+#else
+    typedef gulong window_id_type;
 #endif
```

Comment:
I think this should be moved to `wx/gtk/private/mediactrl.h` too. And ideally it ought to be removed to `wxGtkWindowIdType` or something like this, to make it clear that it's something wx-specific.

Reply:
actually this is not GTK specific: in wxGStreamerMediaBackend::SetupXOverlay(), the windows_id is used even without GTK